### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by xPaw**
I went through all the changelogs, and have not noticed any breaking changes that we need to account for.

### CODE REVIEW

**Comment by maxpoulin64** [comment]
Looks good to me, 👍

File: package.json [hunk 1]
```diff
@@ -32,25 +32,24 @@
...
     "express": "4.13.4",
-    "lodash": "4.6.1",
     "irc-framework": "1.0.10",
+    "lodash": "4.11.1",
     "mkdirp": "0.5.1",
-    "moment": "2.12.0",
+    "moment": "2.13.0",
     "read": "1.0.7",
-    "request": "2.69.0",
-    "slate-irc": "0.8.1",
```
Comment (xPaw) [inline package.json]: Left over.

File: package.json [hunk 2]
```diff
@@ -32,25 +32,23 @@
...
+    "request": "2.72.0",
     "socket.io": "1.4.5",
-    "spdy": "3.2.3"
+    "spdy": "3.3.2"
   },
   "devDependencies": {
     "chai": "3.5.0",
-    "eslint": "2.4.0",
-    "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
```
Comment (xPaw) [inline package.json]: Removed as it's installed by grunt either way. http://gruntjs.com/blog/2016-02-11-grunt-1.0.0-rc1-released#npm-scripts

**Comment by astorije** [comment]
👍 and merging.

**Pull request merged**